prawcore/rate_limit.py

```diff
@@ -16,13 +16,13 @@ class RateLimiter(object):
 
     """
 
-    def __init__(self) -> None:
+    def __init__(self, *args, window_size: int) -> None:
```

Comment:
I missed this before, but what is `*args` for? If you're making it so that `window_size` is keyword only you can do this instead:

```suggestion
    def __init__(self, *, window_size: int) -> None:
```

Reply:
I think I just copied that from somewhere else in the code